.github/publish-to-pypi.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Python distributions to PyPI and TestPyPI
```

Comment:
I'm not sure this workflow is valid for a package that's not pure Python. We should probably be using cibuildwheels (as we now do in yt itself) to compile wheels on different platforms instead of just ubuntu.